chain/network/src/routing.rs

```diff
@@ -797,16 +795,6 @@ impl RoutingTable {
                 true
             }
         });
-        if force_pruning {
-            let mut edges_to_remove2 = vec![];
-            for (k, edge) in self.edges_info.iter() {
-                if !self.peer_last_time_reachable.contains_key(&k.0)
-                    || !self.peer_last_time_reachable.contains_key(&k.1)
-                {
-                    edges_to_remove2.push(edge.clone());
-                }
-            }
-        }
```

Comment:
why is this piece of code removed?

Reply:
This code is only executed during tests, and it isn't needed any more.